app/README.md

```diff
@@ -30,13 +30,14 @@ For linting, this application is leveraging `eslint`, `prettier` and Nava's [esl
 
 In VSCode, do so by creating a `.vscode/settings.json` file with:
 
-```
+```json
 {
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
```

Comment:
this still means that format on save is done by prettier not eslint?

Reply:
Yep, I think this `fixAll` line results in the `--fix` flag being ran with ESLint when a file is changed: https://eslint.org/docs/latest/user-guide/command-line-interface#fixing-problems